lib/configure.js

```diff
@@ -40,6 +40,8 @@ module.exports = async () => {
   const lintIgnorePatterns = [
     addSep(bundleReportFolder),
     addSep(coverageFolder),
+    '*.css.js.d.ts',
+    '*.less.d.ts',
```

Comment:
Ignoring the linting of our generated definition files as they clash with prettier and there is no real value here as we programmatically generate them anyway.